chemist/src/main/scala/flask.scala

```diff
@@ -48,23 +50,44 @@ object Flask {
    * Given a collection of flask instances, find out what exactly they are already
    * mirroring and absorb that into the view of the world.
    *
-   * This function should only really be used startup of chemist.
+   * If some flasks are down then they will not be returned as part of distribution
+   * and should not be allocated work too. This might lead to double work assignment in case of
+   * network partitioning but Chemist need to be able to correct double assignment issues anyways.
+   *
+   * This function should only really be used startup of chemist. //wrong??? part of discovery and it is periodic
```

Comment:
Yeah, this comment can go away. You're right that it is indeed used more than just startup (that was the old behaviour)